WordPress/Classes/ViewRelated/Reader/WPImageViewController.m

```diff
@@ -102,8 +109,10 @@ - (void)viewDidLoad
     [tgr1 requireGestureRecognizerToFail:tgr2];
     [self.scrollView addGestureRecognizer:tgr1];
 
-    self.flingableViewHandler = [[FlingableViewHandler alloc] initWithTargetView:self.scrollView];
-    self.flingableViewHandler.delegate = self;
+    if (!self.galleryMode) {
```

Comment:
Since `galleryMode` is only flagging whether the `FlingableViewHandle` should be used, it might be nice to rename it according to how its used.  Then if we extend `WPImageViewController` in the future and add more `init`s, the flag could be reused in a new context.  Make sense?